registry/auth/token/accesscontroller.go

```diff
@@ -302,6 +304,29 @@ func getSigningAlgorithms(algos []string) ([]jose.SignatureAlgorithm, error) {
 	return signAlgVals, nil
 }
 
+// Generate and return a 'libtrust' fingerprint of the public key.
+// For an RSA key this should be:
+//   SHA256(DER encoded ASN1)
+// Then truncated to 240 bits and encoded into 12 base32 groups like so:
+//   ABCD:EFGH:IJKL:MNOP:QRST:UVWX:YZ23:4567:ABCD:EFGH:IJKL:MNOP
+func libtrustKeyIDFromPublicKey(pubKey any) string {
```

Comment:
the issue we had with `libtrust` is it does things that are not standardized anywhere like this ID generation func.

That was actually the main reason we went for explicit JWKs defined on the consumer side.

Reply:
The intention to introduce breaking changes should be guided by the goal of minimizing disruption. For instance, when implementing breaking changes in Go, there are multiple phases such as opt-in, opt-out, and eventual removal.

I agree that non-standard key id generation like this is not good. But `libtrust` is already gone, the key id generation in this PR is more likely mental burden instead of maintainence burden. A phased deprecation plan with decent documentation or migration guidance would be helpful to the community.

PS: The document still states that `rootcertbundle` is a required configuration option. If I understand correctly, it should be optionally after the introduce of `jwks`.

Reply:
Cute. Thanks for the lecture on b/w compatibility. I appreciate your suggestions here. One way of getting your changes into this project is by becoming a more active contributor. 

As I said I am personally not in favour of this change but maybe other maintainers will be. So 🤞 

> PS: The document still states that rootcertbundle is a required configuration option. If I understand correctly, it should be optionally after the introduce of jwks.

Yep, the docs need a proper review; I suspect there is more outdated info in them, unfortunately.